### ISSUE TITLE
If the name includes path separators, openssl fails

### ISSUE THREAD

**Issue by toddself** [davewasmer/devcert#6]
If you pass in a name that contains your system's path separators it fails because the directory doesn't exist for openssl to write to, since it's a shell command:

```
{ Error: Command failed: openssl genrsa -out /Users/todd/.config/devcert/Users/todd/src/bankai/example.js.key 2048
    at checkExecSyncError (child_process.js:481:13)
    at Object.execSync (child_process.js:521:13)
    at run (/Users/todd/src/bankai/node_modules/devcert/dist/utils.js:20:28)
    at openssl (/Users/todd/src/bankai/node_modules/devcert/dist/utils.js:10:12)
    at Object.generateKey (/Users/todd/src/bankai/node_modules/devcert/dist/utils.js:33:5)
    at /Users/todd/src/bankai/node_modules/devcert/dist/index.js:39:21
    at next (native)
    at /Users/todd/src/bankai/node_modules/tslib/tslib.js:96:75
    at Object.__awaiter (/Users/todd/src/bankai/node_modules/tslib/tslib.js:92:16)
    at devcert (/Users/todd/src/bankai/node_modules/devcert/dist/index.js:23:20)
  error: null,
  cmd: 'openssl genrsa -out /Users/todd/.config/devcert/Users/todd/src/bankai/example.js.key 2048',
  file: '/bin/sh',
  args:
   [ '/bin/sh',
     '-c',
     'openssl genrsa -out /Users/todd/.config/devcert/Users/todd/src/bankai/example.js.key 2048' ],
```

The problem here is that devcert created `/Users/todd/.config/devcert` but then the key "name" doesn't exist as a path.

**Comment by toddself** [comment]
Fixed in #7

**Comment by davewasmer** [comment]
This is fixed in the refactor that has landed in master.
